### PR TITLE
Update version to 0.6.0

### PR DESCRIPTION
**Pull request by jerqi**

### CODE REVIEW

**Comment by jerqi** [comment]
@jerryshao @xunliu @yuqi1129  Could you help me review this pull request?

File: init/spark/init.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@
 mkdir -p /opt/spark/conf
 cp /tmp/spark/spark-defaults.conf /opt/spark/conf
 wget https://repo1.maven.org/maven2/org/apache/iceberg/iceberg-spark-runtime-3.4_2.12/1.5.2/iceberg-spark-runtime-3.4_2.12-1.5.2.jar  -O /opt/spark/jars/iceberg-spark-runtime-3.4_2.12-1.5.2.jar
-wget https://repository.apache.org/content/repositories/orgapachegravitino-1090/org/apache/gravitino/gravitino-spark-connector-runtime-3.4_2.12/0.6.0-incubating/gravitino-spark-connector-runtime-3.4_2.12-0.6.0-incubating.jar -O /opt/spark/jars/gravitino-spark-connector-runtime-3.4_2.12-0.6.0-incubating.jar
+wget https://repository.apache.org/content/repositories/staging/org/apache/gravitino/gravitino-spark-connector-runtime-3.4_2.12/0.6.0-incubating/gravitino-spark-connector-runtime-3.4_2.12-0.6.0-incubating.jar -O /opt/spark/jars/gravitino-spark-connector-runtime-3.4_2.12-0.6.0-incubating.jar
```
Comment (yuqi1129) [inline init/spark/init.sh:22]: Is it a general action to download jars from the apache staging repository?

Comment (jerqi) [reply init/spark/init.sh:22]: I copy it from official website. You can see
https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-spark-connector-runtime-3.4

Comment (yuqi1129) [reply init/spark/init.sh:22]: I see.

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**